docs/userguide/networking/get-started-overlay.md

```diff
@@ -80,26 +77,32 @@ Keep your terminal open and move onto the next step.
 ## Step 2: Create a Swarm cluster
 
 In this step, you use `docker-machine` to provision the hosts for your network.
-At this point, you won't actually created the network. You'll create several
-machines in VirtualBox. One of the machines will act as the Swarm master;
-you'll create that first. As you create each host, you'll pass the Engine on
-that machine options that are needed by the `overlay` network driver.
+At this point, you won't actually be creating the network. You'll create several
+machines in VirtualBox. One of the machines will act as the Swarm master; you'll
+create that first. As you create each host, you'll pass the created Docker
+Engine on that configuration options that are needed by the `overlay` network
+driver.
 
 1. Create a Swarm master.
 
 			$ docker-machine create \
-			-d VirtualBox \
+			-d virtualbox \
 			--swarm --swarm-image="swarm" --swarm-master \
 			--swarm-discovery="consul://$(docker-machine ip mh-keystore):8500" \
-			--engine-opt="cluster-store=consul://$(docker-machine ip mh-keystore):8500"
+			--engine-opt="cluster-store=consul://$(docker-machine ip mh-keystore):8500" \
 			--engine-opt="cluster-advertise=eth1:2376" \
 			mhs-demo0
 
-	At creation time, you supply the Engine `daemon` with the ` --cluster-store` option. This option tells the Engine the location of the key-value store for the `overlay` network. The bash expansion `$(docker-machine ip mh-keystore)` resolves to the IP address of the Consul server you created in "STEP 1". The `--cluster-advertise` option advertises the machine on the network.
+	At creation time, you supply the Engine `daemon` with the ` --cluster-store`
+option. This option tells the Engine the location of the key-value store for the
+`overlay` network. The bash expansion `$(docker-machine ip mh-keystore)`
+resolves to the IP address of the Consul server you created in "STEP 1". The
+`--cluster-advertise` option advertises the machine on the network.
```

Comment:
Can you indent these lines to match the first line in the paragraph (to prevent breaking the layout if blank lines are introduced at some point)